src/Bicep.Core/Semantics/BindableSymbol.cs

```diff
@@ -15,5 +15,4 @@ protected BindableSymbol(string name, IdentifierSyntax nameSyntax) : base(name)
         /// </summary>
         public IdentifierSyntax NameSyntax { get; }
```

Comment:
I believe this class was created purely to facilitate the duplicated binders - is it still necessary?
<img width="734" alt="image" src="https://user-images.githubusercontent.com/38542602/194467787-b3f349e5-c9f4-4320-b603-f5e5b1433763.png">

Reply:
nope - will remove